src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.interfaces.WriterService;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class WriterServiceImpl implements WriterService {
+    private static final String FILE_PATH = "dataTxt/report.txt";
+
+    @Override
+    public void writeToFile(String data) {
+        File file = new File(FILE_PATH);
```

Comment:
Path need to be defined in method writeToFile() method
```suggestion
    public void writeToFile(String filepath, String data) {
        File file = new File(Path.of(filepath));
```